.codespell/README.md

```diff
@@ -0,0 +1,74 @@
+# Codespell Configuration
```

Comment:
IMO, this contains a lot of valuable information that might be better moved to a docs page (linked to from this `README.md` file) than here where it is not searchable.